database/migrations/2022_10_19_014325_create_categories_table.php

```diff
@@ -2,6 +2,7 @@
 
 use Illuminate\Database\Migrations\Migration;
 use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Seeder;
```

Comment:
I recall Andrew saying that we shouldn't need to change the migrations but I think this is fine anyway!

Reply:
this is not needed, should be removed